IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CommunicationsApplicanceTypeAutomaton/DocPropertySet.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CommunicationsApplicanceTypeAutomaton" UniqueId="ac8ba82e-d39e-0184-a856-fa356c131fe6" ApplicableType="IfcCommunicationsAppliance/AUTOMATON" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="RatedVoltage_0_oie0qSSHuO00025QrEV" />
+		<DocProperty xsi:nil="true" href="InputSignalType_3mt7oNhWL3QuPz1Q2ybJCo" />
+		<DocProperty xsi:nil="true" href="OutputSignalType_2TKPRtqQfDlgGubE0HD4kt" />
+	</Properties>
+</DocPropertySet>
+
```

Comment:
Rename => "Pset_CommunicationsApplianceTypeAutomation" (include an "i")?
Documentation missing.

Reply:
Maybe it is supposed to be applicance?

Reply:
@czapplitec  Right